Database/Base/ShardBase.sql

```diff
@@ -313,251 +245,168 @@ CREATE TABLE `ace_position` (
   `qX` float NOT NULL,
   `qY` float NOT NULL,
   `qZ` float NOT NULL,
+  `landblock` int(5) unsigned GENERATED ALWAYS AS (conv(left(hex(`landblockRaw`),4),16,10)) STORED,
+  `cell` int(5) unsigned GENERATED ALWAYS AS (conv(lpad(substr(hex(`landblockRaw`),5,4),4,'0'),16,10)) STORED,
   PRIMARY KEY (`positionId`),
   KEY `idx_aceObjectId` (`aceObjectId`),
-  KEY `idx_landblock` (`landblockRaw`),
   KEY `idxPostionType` (`positionType`),
-  CONSTRAINT `fk_ap_ao` FOREIGN KEY (`aceObjectId`) REFERENCES `ace_object` (`aceObjectId`)
-) ENGINE=InnoDB AUTO_INCREMENT=43247 DEFAULT CHARSET=utf8;
-/*!40101 SET character_set_client = @saved_cs_client */;
-
---
--- Table structure for table `character_friends`
---
-
-DROP TABLE IF EXISTS `character_friends`;
-/*!40101 SET @saved_cs_client     = @@character_set_client */;
-/*!40101 SET character_set_client = utf8 */;
-CREATE TABLE `character_friends` (
-  `id` int(10) unsigned NOT NULL,
-  `friendId` int(10) unsigned NOT NULL,
-  PRIMARY KEY (`id`,`friendId`),
-  KEY `FK_character_friends_friendId_character_guid` (`friendId`)
-  /*
-  CONSTRAINT `FK_character_friends_friendId_character_guid` FOREIGN KEY (`friendId`) REFERENCES `character` (`guid`),
-  CONSTRAINT `FK_character_friends_id_character_guid` FOREIGN KEY (`id`) REFERENCES `character` (`guid`)
-  */
-) ENGINE=InnoDB DEFAULT CHARSET=utf8;
-/*!40101 SET character_set_client = @saved_cs_client */;
+  KEY `idx_landblock_raw` (`landblockRaw`),
+  KEY `idx_landblock` (`landblock`),
+  KEY `idx_cell` (`cell`),
+  CONSTRAINT `fk_position_ao` FOREIGN KEY (`aceObjectId`) REFERENCES `ace_object` (`aceObjectId`) ON DELETE CASCADE
+) ENGINE=InnoDB AUTO_INCREMENT=43262 DEFAULT CHARSET=utf8;
```

Comment:
I found out how that happened and I fixed it but it is a FYI note to anyone doing a schema export - it you have data - and any of your tables use auto_increment - then it takes the highest current pk value in your db as the seed.   You either need to export schema from a empty db or remember to fix that.  just FYI